doc/nrf/config_and_build/dfu/dfu_image_versions.rst

```diff
@@ -36,7 +36,10 @@ This option can refer to two different things:
 Configuring image version with MCUboot
 **************************************
 
-To assign a semantic version number to your application when you have opted for booting application directly by the MCUboot bootloader (that is, if you have opted for either :ref:`ug_bootloader_adding_immutable_mcuboot` or :ref:`ug_bootloader_adding_upgradable_mcuboot`), add the version string to the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` option for the application:
+To assign a semantic version number to your application when you have opted for booting application directly by the MCUboot bootloader (that is, if you have opted for either :ref:`ug_bootloader_adding_immutable_mcuboot` or :ref:`ug_bootloader_adding_upgradable_mcuboot`), it is recommended to use the :file:`VERSION` file that contains the version information for the application image.
```

Comment:
I suggest that we link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/build/version/index.html